metadata/network.go

```diff
@@ -23,30 +23,33 @@ import (
 
 // NetworkDefinition structure holds all parameters which describe particular network
 type NetworkDefinition struct {
-	DiscoveryAPIAddress     string
-	BrokerAddress           string
-	EtherClientRPC          string
-	QualityOracle           string
-	PaymentsContractAddress common.Address
+	DiscoveryAPIAddress               string
+	AccessPolicyOracleEndpointAddress string
```

Comment:
Hm, this is not used anywhere? It will probably be used in https://github.com/mysteriumnetwork/node/issues/865 (using `trust-oracle` instead of using hardcoded rules) ?